client/src/components/CheckInAndOut.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Guests from './Guests.jsx';
+import CheckInCal from './CheckInCal.jsx';
+class CheckInAndOut extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      checkIn_clicked: false,
+      checkIn_picked: false
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    e.preventDefault();
+    // let check_in_date =
+    // this.props.submit(check_in_date)
+  }
```

Comment:
This doesn't appear to be doing any thing at the moment. Be sure to update as you go!

Reply:
it's there for the future! sorry to be confusing